dir.c

```diff
@@ -3501,10 +3506,14 @@ dir_s_home(int argc, VALUE *argv, VALUE obj)
 #if 0
 /*
  * call-seq:
- *   Dir.exist?(file_name)   ->  true or false
+ *   Dir.exist?(dirpath) ->  true or false
+ *
+ * Returns whether +dirpath+ is a directory in the underlying file system:
+ *
+ *   Dir.exist?('/example')         # => true
+ *   Dir.exist?('/nosuch')          # => false
+ *   Dir.exist?('/example.main.rb') # => false
```

Comment:
Was the period a typo?

```suggestion
 *   Dir.exist?('/example/main.rb') # => false
```